src/client/components/Login.tsx

```diff
@@ -23,16 +26,36 @@ export const Login: React.FC<LoginProps> = () => {
     }
 
     return(
+        
+
         <div>
-            <h3>Login</h3>
-            <Card>
-                <CardContent>
-                    <TextField label="User Name" onChange={onTextFieldUpdate(setUserName)}></TextField>
-                    <Button onClick={()=>{printData(userName, password)}}>
-                        Test
-                    </Button>
-                </CardContent>
-            </Card>
+
+            <Paper square>
+                    <Card>
+                            <CardHeader title='Welcome Back!' />
+                            <CardContent>
+                                <Tabs
+                                    value={userName}
+                                    indicatorColor="primary"
+                                    textColor="primary"
+                                    onChange={onTextFieldUpdate(setUserName)}>
+                                    <Tab label="Login" />
+                                    <Link href='/signup'><Tab label="Sign Up" /></Link>
+                                </Tabs>
+                            </CardContent>
+                            <CardContent><TextField label="User Name" onChange={onTextFieldUpdate(setUserName)}></TextField></CardContent>
```

Comment:
<CardContent>
    <TextField label="User Name" onChange={onTextFieldUpdate(setUserName)} />
</CardContent>

 is cleaner and concise